src/idl_gen_general.cpp

```diff
@@ -1023,6 +1036,26 @@ void GenStruct(StructDef &struct_def, std::string *code_ptr) {
       code += lang_.getter_suffix;
       code += "}\n";
     }
+	// Generate ByKey accessor for vector of struct with key
+	if (field.value.type.base_type == BASE_TYPE_VECTOR &&
```

Comment:
How does this relate to the current PR? If this was a fix, could this have been made into a seperate PR that can be approved seperately? Smaller PRs for individual items are easier and quicker to review.